example/tls_ws_both.cpp

```diff
@@ -365,7 +365,7 @@ int main(int argc, char** argv) {
     std::uint16_t pid_sub2;
 
     auto c = MQTT_NS::make_tls_sync_client_ws(ioc, "localhost", port);
-    c->set_ca_cert_file(base + "cacert.pem");
+    c->get_context().load_verify_file(base + "cacert.pem");
```

Comment:
I recommend

`get_ssl_context()`

So that it does not get confused with boost::asio::io_context()

Reply:
Indeed. I will update it.
P.S. I personally don't like the word `ssl` but like `tls`. But keeping asio manner is better for that.